### ISSUE TITLE
Warn on same export

### ISSUE THREAD

**Issue by TrySound** [rollup/rollup#722]
Would be great to warn if there is same exports in module. This example is not explicit and could be dangerous.
```js
// mod1
export const param = 0;
// mod2
export const param = 1;
// mod3
export * from './mod1';
export * from './mod2';
```

**Comment by eventualbuddha** [comment]
To be clear, this is allowed by the spec and has well-defined semantics. However, you're right that if could indicate a bug. Warning sounds reasonable.

**Comment by Rich-Harris** [comment]
Released as 0.33
